src/vip_client/classes/VipLauncher.py

```diff
@@ -49,7 +49,7 @@ class VipLauncher():
     # Default file name to save session properties 
     _SAVE_FILE = "session_data.json"
     # Vip portal
-    _VIP_PORTAL = "https://vip.creatis.insa-lyon.fr/"
+    _VIP_PORTAL = "https://vip.creatis.insa-lyon.fr/rest/"
```

Comment:
actually the URL is used at various places as link of the portal, so it should not include "rest" at the end.
Juste add "rest" to it when you create the vip-client